### PR TITLE
Add missing fields

### PR DESCRIPTION
**Pull request by tiago-loureiro**
Fields 5 and 6 were missing, 7 is the new field. @typfel / @fisx

### CODE REVIEW

File: proto/otr.proto [hunk 1]
```diff
@@ -21,10 +21,18 @@ message UserEntry {
...
+      LOW_PRIORITY = 1;
+      HIGH_PRIORITY = 2;
+    };
     required ClientId  sender      = 1;
     repeated UserEntry recipients  = 2;
     optional bool      native_push = 3 [default = true];
     optional bytes     blob        = 4;
+    optional Priority  native_priority = 5;
+    optional bool      transient   = 6;
+    optional UserId    report_missing = 7;
```
Comment (typfel) [inline proto/otr.proto]: This should be a list?

Comment (tiago-loureiro) [inline proto/otr.proto]:
```suggestion
    repeated UserId    report_missing = 7;
```

Comment (tiago-loureiro) [reply proto/otr.proto]: Commented on my own PR :)

Comment (tiago-loureiro) [reply proto/otr.proto]: (Thanks!)

**Pull request merged**